### ISSUE TITLE
No output on monitor/screen

### ISSUE THREAD

**Issue by DerpyJakey** [tolga9009/elgato-gchd#16]
I follow the instructions. As soon as I use VLC or OBS to watch the input the monitor turns off. But successfully displays on VLC/OBS

**Comment by tolga9009** [comment]
I'm awfully sorry for overlooking this issue! Just spotted it, cause another issue got reported.

If the problem is still persistent, my wild guess would be, that your TV probably doesn't support either the resolution or the colorspace. But that sounds very unlikely to be honest. Can you give me more details about the issue?

Which console / resolution / TV are you using? And can you give me the full command-line arguments, you're using?

Cheers,
Tolga

**Comment by DerpyJakey** [comment]
No need to apologize! :D And it turned out it was my monitor. Tested it on another monitor. But there's still a problem. Both the computer (OBS + VLC) and Monitor shows my ps4 in the wrong color. Its all Green.
![screenshot_20160428_225405](https://cloud.githubusercontent.com/assets/8509775/14907871/49c93ac4-0d94-11e6-9b9d-fc65c4afe82f.png)

**Comment by bloodywing** [comment]
This is issue #8

**Comment by DerpyJakey** [comment]
Yep. Sorry I feel stupid. xP Anyway thanks for all your help.

**Comment by tolga9009** [comment]
No, you're helping me to improve this driver with reporting bugs. Thanks for your report, no need to feel stupid - it was a different bug. Still, I'm wondering why your monitor didn't show any picture. Could be unsupported colorspace, unsupported resolution or maybe something else.

Either way, I think once issue #8 and #14 are fixed, your original issue should be fixed aswell. Still, I'm leaving this closed, as it seems to be a mix of multiple issues, which have already been filed.

Thank you!

**Comment by tolga9009** [comment]
I'm reopening this issue, since I have encountered exactly the same with my current setup. My setup:
Dell T20 + GTX750 Ti HDMI Out -> Elgato GCHD HDMI In -> HDMI-to-DVI cable -> Dell HP ZR22w 1920x1080p monitor, operating system is configured to 1920x1080 resolution. Once I run the capture on Linux, the screen turns black and doesn't recover, but I'm able to capture it on my Linux machine.

This only happens, when I'm using a HDMI-to-DVI cable. Using a normal HDMI cable "fixes" it. However, this use case should also be covered, I will work on this aswell.

**Comment by tolga9009** [comment]
The driver has been completely overhauled, I think this needs to be tested and reevaluated. If anyone has a HDMI-to-DVI cable, please test and report.

**Comment by smeinecke** [comment]
Hi, tested this today and the problem still persists with HDMI=>DVI Cable. As soon as I try to capture anything the display only shows "Input not supported". Latest git repo checkout + 3.2 firmware is used.
